overview/default.nix

```diff
@@ -319,12 +319,36 @@ let
             }
           ];
         };
+        # TODO: this will be refactored
```

Comment:
Yes but how ;)

Reply:
Like this: https://github.com/ngi-nix/ngipkgs/pull/1170

I've had this for a while as you were aware, but I couldn't find a nice and clean way to re-use the env var in CI, so let's just leave that for later.